.tekton/numaresources-operator-4-20-push.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/nudge_renovate_config_map: namespace-wide-nudging-renovate-config
+    build.appstudio.openshift.io/repo: https://github.com/openshift-kni/numaresources-operator?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/cancel-in-progress: "false"
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: |
+      event == "push" &&
+      target_branch == "main" &&
+      (
+        '.tekton/build-pipeline.yaml'.pathChanged() ||
+        '.konflux/container-build.args'.pathChanged() ||
+        'api/***'.pathChanged() ||
+        'cmd/***'.pathChanged() ||
+        'config/***'.pathChanged() ||
+        'hack/***'.pathChanged() ||
+        'internal/***'.pathChanged() ||
+        'must-gather/***'.pathChanged() ||
+        'nrovalidate/***'.pathChanged() ||
+        'pkg/***'.pathChanged() ||
+        'rte/***'.pathChanged() ||
+        'tools/***'.pathChanged() ||
+        'vendor/***'.pathChanged() ||
+        'go.mod'.pathChanged() ||
+        'go.sum'.pathChanged() ||
+        '.tekton/numaresources-operator-4-20-push.yaml'.pathChanged() ||
+        '.konflux/operator/***'.pathChanged()
+      )
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: numaresources-operator-4-20
+    appstudio.openshift.io/component: numaresources-operator-4-20
+    pipelines.appstudio.openshift.io/type: build
+  name: numaresources-operator-4-20-on-push
+  namespace: telco-5g-tenant
+spec:
+  params:
+  - name: git-url
+    value: '{{source_url}}'
+  - name: revision
+    value: '{{revision}}'
+  - name: output-image
+    value: quay.io/redhat-user-workloads/telco-5g-tenant/numaresources-operator-4-20:{{revision}}
+  - name: build-platforms
```

Comment:
I recommend adding the lastest tag to the image produced on push, it'll make it easier to identify the latest image pre-stage image

Reply:
same applies to all on-push pipelines for different components

Reply:
will consider it in a different PR if it does not break bundles.